geode-core/src/main/java/org/apache/geode/distributed/internal/ReplyMessage.java

```diff
@@ -295,7 +296,14 @@ public void toData(DataOutput out,
       out.writeInt(processorId);
     }
     if (this.returnValueIsException || this.returnValue != null) {
-      DataSerializer.writeObject(this.returnValue, out);
+      try {
+        DataSerializer.writeObject(this.returnValue, out);
```

Comment:
toData/fromData methods in this class could use SerializationContext/DeserializationContext instead of DataSerializer.

Reply:
Good idea - I've pushed an update to do that